models/settings.go

```diff
@@ -27,6 +27,11 @@ type MetricsResolutions struct {
 	LR time.Duration `json:"lr"`
 }
 
+// QAN contains query analytics configuration.
+type QAN struct {
+	DataRetention uint32 `json:"data_retention"` // number of days data retention
```

Comment:
Please use `time.Duration`

Reply:
@AlekSi, just to confirm. The minimum truncation of partitions is a day. So is it OK, from UX point of view, to specify data retention as a `time.Duration` where maximum is an hour. In this case user or UI need to multiply days by 24 and on api side we need to do this in a reverse order.

Reply:
API will use standard type `google.protobuf.Duration` so we don't use several different types in the same API for durations. But there should be a validation that incoming value represents X whole days.
Datastore will use standard type `time.Duration` for the same reason. There also should be a validation.
In the future, that will allow us to reduce granularity (say, to hours instead of days) without breaking backward compatibility of API and stored data.

> time.Duration where maximum is an hour
`time.Hour` is a maximum predefined constant, but nothing prevents us from using `720*time.Hour` to represent 30 days.